### PR TITLE
Turn into a python package

### LINKED ISSUE #56: Add to PyPi for easy install

### PR DESCRIPTION
**Pull request by bwbeach**
Fix #56 

I'm new at creating Python packages.  Suggestions for improvements are welcome.

### CODE REVIEW

**Comment by mathiasrw** [comment]
This is perfect - one step closer to be able to backup postgres databases to B2 with https://github.com/wal-e/wal-e

File: pre-commit.sh [hunk 1]
```diff
@@ -9,34 +9,44 @@ function header
...
-else
-    echo "Formatting updated:"
-    echo
-    diff b2 b2.yapf.out
-    mv b2.yapf.out b2
-    chmod +x b2
-fi
+for src_file in $SOURCE_FILES
+do
+    echo $src_file
```
Comment (ppolewicz) [inline pre-commit.sh]: Please **always** quote strings in bash. Otherwise random stuff stops working when a space appears in a variable or a file name.

**Comment by ppolewicz** [comment]
It looks good (except bash variable quoting).

Installation instructions should probably be adjusted to reflect this change.

Please merge it soon (renaming the main file could conflict with everything)

File: pre-commit.sh (hunk 1 above)
Comment (bwbeach) [inline pre-commit.sh]: I'll fix the quoting.  Thanks.

**Pull request merged**